src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fuse_transpose_reorder.cpp

```diff
@@ -45,7 +45,7 @@ void FuseTransposeAndReorderTest::CheckTransposeCount(size_t expectedTransposeCo
 
 void FuseTransposeAndReorderTest::SetUp() {
     targetDevice = CommonTestUtils::DEVICE_CPU;
-
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
```

Comment:
Why do we add this? Is there a case skipped?

Reply:
There are some transpose test failures:
![image](https://github.com/openvinotoolkit/openvino/assets/31196718/24a25ee8-7f5d-4d1f-a5a2-8b18d584789a)

The reason is that preProcess is added before transformation in case of legacy public api + plugin api 2.0, which cause original `transpose+reorder` subgraph test become `convert + transpose + reorder` subgraph and transformation will output below result:

![image](https://github.com/openvinotoolkit/openvino/assets/31196718/d8a18c85-d45d-462a-8050-77c84be028f7)


While this test will check whether transpose still exists in the compiled_model, so it failed.
We have to skip this test case, so add this` SKIP_IF_CURRENT_TEST_IS_DISABLED()` to skip it in the skip lists.

Reply:
The skip check is performed in scope of Run() function, before the check for a transpose node happens.
Why do we have to do it in SetUp()?

Reply:
yes, you are right, skip checking is performed in Run(), but after Run() it still will check test result, so we have to skip it in SetUp() to avoid this problem.
![image](https://github.com/openvinotoolkit/openvino/assets/31196718/4584c9e2-67c4-4d9f-bdcf-4b554bc42e36)